src/scripts/getConnectathonBundles.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
+const path = require('path');
+const mongoUtil = require('../util/mongo');
+const { createResource } = require('../util/mongo.controller');
+
+const connectathonPath = path.resolve(path.join(__dirname, '../../connectathon/fhir401/bundles/measure/'));
+// bundles that are not the latest available version
+const IGNORED_BUNDLES = ['EXM347-3.2.000-bundle.json'];
+// files containing EXM bundles of interest from connectathon repo
+const bundleFiles = [];
+
+/**
+ * Retrieves EXM bundle files from the connectathon repo using a regular expression.
+ * Uses recursion to parse through all available subdirectories.
+ * @param {string} directory - directory path to start at
+ * @returns array of string paths that represent the bundle files of interest
+ */
+const getBundleFiles = directory => {
+  const fileNameRegExp = new RegExp(/(^EXM.*.json$)/);
+  const filesInDirectory = fs.readdirSync(directory);
+  filesInDirectory.forEach(file => {
+    const absolute = path.join(directory, file);
+    if (fs.statSync(absolute).isDirectory()) {
+      getBundleFiles(absolute);
+    } else if (fileNameRegExp.test(file) && !IGNORED_BUNDLES.includes(file)) {
+      bundleFiles.push(absolute);
+    }
+  });
+  return bundleFiles;
+};
+
+/**
+ * Uploads all the resources from the connectathon measure bundles into the
+ * database.
+ *
+ * TODO: Currently configured for fhir401 connectathon measure bundles,
+ * but may want to expand to other measure bundle providers in the future.
+ */
+async function main() {
+  const bundleFiles = getBundleFiles(connectathonPath);
```

Comment:
I think it's a little confusing to overwrite this global variable here. Consider removing the return statement from getBundleFiles and then just calling getBundleFiles here and accessing the global bundleFiles variable which should be the same.

Reply:
Good catch, I think I was renaming stuff and didn't realize I did this